util/global_community_silence.php

```diff
@@ -36,11 +36,11 @@
 use Friendica\Database\DBM;
 use Friendica\Network\Probe;
 
-require_once'"boot.php');
-require_once('include/dba.php');
-require_once('include/text.php');
+require_once 'boot.php';
+require_once 'include/dba.php';
+require_once 'include/text.php';
 $a = get_app();
-require_once('.htconfig.php');
+require_once'.htconfig.php';
```

Comment:
Not sure if a space is needed after require_once but we have been adding one so far.